src/v/redpanda/admin/security.cc

```diff
@@ -751,3 +750,26 @@ admin_server::list_roles_handler(std::unique_ptr<ss::http::request> req) {
 
     return ss::make_ready_future<ss::json::json_return_type>(j_res);
 }
+
+ss::future<ss::json::json_return_type>
+admin_server::get_role_handler(std::unique_ptr<ss::http::request> req) {
+    ss::sstring role_v;
```

Comment:
The request will get routed to a different admin API endpoint if it is empty. But otherwise, I don't see any problems with `role` being empty here. Am I missing something?

Reply:
No I forgot the router sorts this out.